packages/indexer-client/src/IndexerClient.ts

```diff
@@ -480,7 +480,11 @@ export class IndexerClient extends IndexerBaseClient {
   /**
    * Get all interest funding payments for a given subaccount with the standard pagination response
    * This is a simple wrapper over the underlying `getInterestFundingPayments` function. Very little
-   * additional processing is needed because the endpoint is well structured for pagination
+   * additional processing is needed because the endpoint is well structured for pagination.
+   *
+   * We are omitting `nextCursor` & `hasMore`. We typically determine that there are no pages if `nextCursor` is null,
```

Comment:
We're not omitting it though?

Anyway - I've updated the comment on `main` now.